collector/plugins/sources/kstate/nodes.go

```diff
@@ -39,6 +39,11 @@ func buildNodeConditions(node *v1.Node, transforms configuration.Transforms, ts
 		copyLabels(node.GetLabels(), tags)
 		tags["status"] = string(condition.Status)
 		tags["condition"] = string(condition.Type)
+		// TODO: remove after testing node alerts
```

Comment:
seems like this was supposed to go away after alert testing ?

Reply:
It is intended to go after acceptance testing, but I can instead add screenshots to the related story in Jira.

Reply:
Another alternative would be to attach the CRD created with this change directly to Jira and revert the change from PR.